src/pages/Landing/Slider/Slider.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useEffect } from 'react';
+import SwiperData from '../Slider/SwiperData';
+import SliderButton from '../Slider/SliderButton';
+
+const Slider = ({
+  swiperList,
+  setSwiperList,
+  currentIndex,
+  setCurrentIndex,
+}) => {
+  useEffect(() => {
+    fetch('data/swiperItems.json')
+      .then(res => res.json())
+      .then(data => {
+        setSwiperList(data);
+      });
+  }, [setSwiperList]);
+
+  function handleSlide(currentIndex) {
+    if (currentIndex === swiperList.length) {
+      currentIndex = 0;
+    } else if (currentIndex < 0) {
+      currentIndex = 4;
```

Comment:
```suggestion
      currentIndex = swiperList.length - 1;
```

4로 하드코딩해두는 것 보다 length - 1을 통해 가장 마지막 아이템을 지정하는게 추후 아이템의 갯수가 바뀌더라도, 코드를 수정할 필요가 없겠네요!